src/Microsoft.Health.Fhir.TemplateManagement/Overlay/OverlayOperator.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Health.Fhir.TemplateManagement.Overlay
 {
     public class OverlayOperator : IOverlayOperator
```

Comment:
Since this is an Operator class, can we simplify the methods to "Extract", "Archive" and "Merge"?

Reply:
They are operations on different type of layers.

Reply:
Yes. 
```Extract``` operation decompresses files from artifacts (bytes)
```Archive``` operation compressed files to artifacts (bytes)
```Merge``` operation merges all file layers

Reply:
Thanks, changed functions name.